notifications/source/source.go

```diff
@@ -0,0 +1,4 @@
+// Package source implements an Icinga Notifications source to send events to the Icinga Notifications API.
+//
+// Start by creating a new Client, which can then submit events to this API.
```

Comment:
> which can then submit events to this API.

I think, you should rephrase this a bit because it's not very clear to me which API it's referring to, or you could drop this entirely as the first sentence is good enough and this adds little value to it.

Reply:
Valid point, thanks. I have reworded this a bit to point users to `Client` and `Client.ProcessEvent` to get started.